webpack.config.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+/**
+ * WordPress dependencies
+ */
+const { config } = require( '@wordpress/webpack-config' );
+
+module.exports = Object.assign(
```

Comment:
Actually, I was thinking that this would be a good default (entry and output) for all projects. I would be glad if projects could make it as simple to use as:
```json
"devDependencies": {
    "@wordpress/scripts": "2.5.0"
},
"scripts": {
    "build": "wp-scripts build"
}
```

Reply:
We should have it up and running by the end of the week. I hope we can publish it to npm soon after.

Reply:
It's now published to npm:

https://unpkg.com/@wordpress/scripts@3.0.0/config/webpack.config.js